instrumentation/armeria-1.0/auto/src/main/java/io/opentelemetry/auto/instrumentation/armeria/v1_0/AbstractArmeriaInstrumentation.java

```diff
@@ -29,7 +29,12 @@ public AbstractArmeriaInstrumentation() {
   @Override
   public String[] helperClassNames() {
     return new String[] {
-      packageName + ".shaded.internal.ContextUtil",
+      packageName + ".shaded.client.ArmeriaClientTracer",
+      packageName + ".shaded.client.ArmeriaClientTracer$ArmeriaSetter",
+      packageName + ".shaded.client.OpenTelemetryClient",
+      packageName + ".shaded.client.OpenTelemetryClient$Decorator",
+      // .thenAccept(log -> lambda
```

Comment:
Not sure if this comment is helping or confusing :)

Reply:
Tried a bit harder